docs/src/language_servers.md

```diff
@@ -512,6 +516,34 @@ There are multiple options:
     }
     ```
 
+### Ruby LSP
+
+1. Install the Ruby LSP gem (see [github:Shopify/ruby-lsp](https://github.com/Shopify/ruby-lsp)):
+
+    ```sh
+    gem install ruby-lsp
+    ```
+
+2. Open `Preferences > Package Settings > LSP > Settings` and add the `"ruby-lsp"` client configuration to the `"clients"`:
+
+    ```jsonc
+    {
+        "clients": {
+            "ruby-lsp": {
+                "enabled": true,
+                "command": ["ruby-lsp"],
+                "selector": "source.ruby | text.html.ruby",
+                "initializationOptions": {
+                    "enabledFeatures": {
+                        "diagnostics": false
+                    },
```

Comment:
Their page says:

> Ruby LSP only supports pull diagnostics

So that's probably why it was disabled but we do support pull diagnostics now so this should work.

Reply:
Took the config from: https://github.com/Shopify/ruby-lsp/blob/main/EDITORS.md#sublime-text-lsp
Generally it works in ST, I tested it.

Reply:
I know but it looks outdated since we support pull diagnostics now.

It might work but likely misses diagnostics which are pretty important part of LSP

Reply:
So we should enable "diagnostics", do I get that right? If so, it mabybe should be done for solargraph, too (personally, I disabled it there because it is to distractive for me).

Reply:
I don't know the reason why diagnostics were disabled in Solargraph configuration. Was there a technical reason or just a personal preference? Unless we have clear reasoning, I would probably leave it as is.

Reply:
I would think that diagnostics should ideally be enabled by default and if someone doesn't like them there is always an option of disabling. Probably better than missing out completely on a potentially useful feature.

Reply:
Changed it for ruby-lsp.